### PR TITLE
[docs] Delete embedded documentation in favor of docs/

### PR DESCRIPTION
**Pull request by ras0219-msft**
After discussion, we've determined that maintaining the documentation inside the port where fixing typos requires republishing the port, is problematic.

This PR performs one final regenerate run (including the missing port helpers `vcpkg-qmake` and `vcpkg-get-python-modules`) and then purges all embedded documentation in the script helpers and port helpers.

This PR does not touch any of the _content_ of this documentation -- that will be done in follow-up PRs.

Drive-by for the helper ports:

1. "documentation": "https://vcpkg.io/en/docs/README.html"
2. "license": "MIT"
3. Use `include_guard(GLOBAL)` in all script files
4. Make sure any persistent variables are saved to the cache

### CODE REVIEW

**Comment by Neumann-A** [comment]
_strictly_ speaking you cannot have the docs of **scripts ports** in `docs/` since the docs probably depend on the **version** which gets installed.

**Comment by ras0219-msft** [comment]
We did discuss that as well (it was one of my concerns).

Our approach moving forward will be to include "Added in version x-y-z" annotations where appropriate in future documentation changes. We won't be "backdating" any existing documentation.

File: docs/maintainers/portfile-functions.md [hunk 1]
```diff
@@ -1,92 +1,16 @@
 <!-- Run regenerate.ps1 to extract scripts documentation -->
 
 # Portfile helper functions
-- [execute\_process](execute_process.md)
```
Comment (BillyONeal) [inline docs/maintainers/portfile-functions.md:4]: This edit seems wrong?

**Comment by BillyONeal** [comment]
Rather than adding an empty regenerate.ps1 can you just delete this block that was calling it?

https://github.com/microsoft/vcpkg/blob/203f5f3bef28ffccb667fdce97bcde7a88aba4e6/scripts/azure-pipelines/windows/azure-pipelines.yml#L49-L56

File: docs/maintainers/portfile-functions.md (hunk 1 above)
Comment (ras0219-msft) [reply docs/maintainers/portfile-functions.md]: Ugh, yeah, that's egg on my face. I just assumed that regenerate would somehow do the right thing after I deleted a bunch of files :)

**Review by BillyONeal — APPROVED** [review]

**Comment by ras0219-msft** [comment]
I'm going to go ahead and merge since the previous [c645bad](https://github.com/microsoft/vcpkg/pull/25096/commits/c645bad940e2478938ef6fd7f14889bce67ef775) passed.

**Comment by dg0yt** [comment]
From draft to world rebuild in less than 24 hours... But I'm glad this documentation update barrier is removed.

**Pull request merged**